### PR TITLE
app-crypt/sops: new package, add 3.9.2

### PR DESCRIPTION
**Pull request by Arian-D**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-crypt/sops/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
```
Comment (parona-source) [inline app-crypt/sops/metadata.xml:4]: Non developers that want to maintain packages need to do it via proxy-maint as such they need to be a co-maintainer in metadatas.

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers

File: app-crypt/sops/sops-3.9.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+DESCRIPTION="Simple And Flexible Tool For Managing Secrets"
+HOMEPAGE="https://getsops.io/"
+SRC_URI="https://github.com/getsops/sops/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/Arian-D/sops-deps/releases/download/${PV}/${P}-vendor.tar.xz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+
+KEYWORDS="amd64 ~arm64"
```
Comment (parona-source) [inline app-crypt/sops/sops-3.9.2.ebuild:15]: You cannot add packages as stable, there is a process for it.

https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages

File: app-crypt/sops/sops-3.9.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module
+
+DESCRIPTION="Simple And Flexible Tool For Managing Secrets"
+HOMEPAGE="https://getsops.io/"
+SRC_URI="https://github.com/getsops/sops/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/Arian-D/sops-deps/releases/download/${PV}/${P}-vendor.tar.xz"
```
Comment (parona-source) [inline app-crypt/sops/sops-3.9.2.ebuild:10]:
```suggestion
SRC_URI="
   https://github.com/getsops/sops/archive/v${PV}.tar.gz
      -> ${P}.tar.gz"
   https://github.com/Arian-D/sops-deps/releases/download/${PV}/${P}-vendor.tar.xz
"
```

matter of taste

File: app-crypt/sops/sops-3.9.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+LICENSE="MPL-2.0"
+SLOT="0"
+
+KEYWORDS="amd64 ~arm64"
+
+src_compile() {
+	ego build || die
```
Comment (parona-source) [inline app-crypt/sops/sops-3.9.2.ebuild:18]: Applies to ego ... || die commands.

https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/go-module.eclass#n208

You are essentially doing `command || die || die`. Meaning its redundant.

Generally all eclass and ebuild commands have || die properly setup already.

**Review by parona-source — COMMENTED** [review]
The main tree generally doesn't take new packages without good reason. You should look into https://wiki.gentoo.org/wiki/Project:GURU

**Comment by Arian-D** [comment]
Just FYI for anyone that comes across this in the future, I added sops in https://github.com/gentoo-mirror/guru/commit/6a0769f2c87384c5cba7c3a1ced122c428ad06e0. If there is any interest for proxy maintaining it in this repo let me know. Otherwise, feel free to use it from GURU.

**Pull request closed without merging**